assets/queries/ansible/aws/ecs_task_definition_container_has_password/query.rego

```diff
@@ -0,0 +1,29 @@
+package Cx
+
+CxPolicy [ result ] {
+	password := ["password", "PASSWORD"]
```

Comment:
can we add more cases like: 'pw' and 'pass'?
and instead of having to create an entry for an all caps word ("PASSWORD"), we could just remove it and apply a lower function when doing the comparison since we already have "password"